website/common/script/content/hatching-potions.js

```diff
@@ -49,44 +49,44 @@ let drops = {
 };
 
 let premium = {
-  Spooky: {
```

Comment:
Whoops, I'll fix that. I didn't notice it in Sublime Text but I can see it in vim.